api/client/model_transformer.go

```diff
@@ -0,0 +1,25 @@
+/*
+ * Merlin
+ *
+ * API Guide for accessing Merlin's model management, deployment, and serving functionalities
+ *
+ * API version: 0.7.0
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+
+package client
+
+import (
+	"time"
+)
+
+type Transformer struct {
+	Enabled         bool             `json:"enabled,omitempty"`
+	Image           string           `json:"image,omitempty"`
+	Command         string           `json:"command,omitempty"`
```

Comment:
The main reason I choose to use string is that I'm using [Text FIeld](https://elastic.github.io/eui/#/forms/form-controls#text-field) instead of [Combo Box](https://elastic.github.io/eui/#/forms/combo-box) as Combo Box doesn't allow us to input the same value more than once. Although not often, there may be a case when we need to specify the same argument twice. So I decide to let the user write the command and args as one string. The splitting from string to an array of string happens at the creation of Transformer spec.